phase/generator_test.go

```diff
@@ -284,8 +284,8 @@ func testGenerator(t *testing.T, when spec.G, it spec.S) {
 							h.AssertContains(t, inputs.TargetEnv,
 								"CNB_TARGET_ARCH=amd64",
 								"CNB_TARGET_ARCH_VARIANT=",
-								"CNB_TARGET_DISTRO_NAME=",
-								"CNB_TARGET_DISTRO_VERSION=",
+								"CNB_TARGET_DISTRO_NAME=ubuntu",
+								"CNB_TARGET_DISTRO_VERSION=22.04",
```

Comment:
```suggestion
```

Reply:
The generator tests are skipped on Windows, which is why these aren't failing, but we don't want to tie ourselves to a particular environment for getting these tests to pass.